targetplatform/EclipseSmartHome.setup

```diff
@@ -455,6 +459,10 @@
           value="true"/>
       <setupTask
           xsi:type="setup:PreferenceTask"
+          key="/instance/org.eclipse.jdt.ui/org.eclipse.jdt.ui.pkgNameAbbreviationPatternForPackagesView"
+          value="org.eclipse.smarthome=o.e.sh"/>
```

Comment:
I'd prefer `o.e.sh` as it still indicates that it is a package abbreviation (similar as logback does it in logfiles), while `esh` might confuse some people.